src/VolumeDrawable.ts

```diff
@@ -48,6 +56,15 @@ export default class VolumeDrawable {
   private fusion: FuseChannel[];
   public sceneRoot: Object3D;
   private meshVolume: MeshVolume;
+  /**
+   * Group for all child objects of the volume. The group is scaled and
+   * transformed to the normalized volume coordinate space, where the volume's
+   * center is at (0, 0, 0) and the bounds have a range of [-0.5, 0.5] along each
+   * axis.
+   */
+  private childObjectsGroup: Group;
+  /** Set of drawable objects that are children of the volume's transform. */
+  private childObjects: Set<IDrawableObject>;
```

Comment:
The `meshVolume` is now a child of the `childObjectsGroup`, and all the transforms that were previously applied to it specifically are now applied to the entire group.